ethereumj-core/src/test/java/org/ethereum/solidity/CompilerTest.java

```diff
@@ -16,17 +16,20 @@
     public void simpleTest() throws IOException {
         String contract =
                 "contract a {" +
-                "  int i1;" +
-                "  function i() returns (int) {" +
-                "    return i1;" +
-                "  }" +
-                "}";
+                        "  int i1;" +
+                        "  function i() returns (int) {" +
+                        "    return i1;" +
+                        "  }" +
+                        "}";
```

Comment:
I made good experience having and auto-format hook before commit. This assures a uniformity of source code format. You can write messy formatted code and it will be committed in a format that many developers agree is a readable format. But I can revert that although I do not think it it’s worse. Adjusted.